liaisons/opcf/tech_reqs.md

```diff
@@ -0,0 +1,23 @@
+# Technical Requirements for OPC UA/WoT Collaboration
+
+## Use Cases
+See https://w3c.github.io/wot-usecases/#use-case-opc-ua-binding
+
+## Objectives
+1. Use OPC UA servers (device, node) from WoT Consumers; treat an OPC UA server as a Thing
+2. Access WoT TD metadata for OPC UA servers - integrate discovery process with OPC UA
+3. ? Let OPC UA servers connect to WoT Things; but if OPC UA devices are ALSO WoT Consumers, 1 satisfies this; --> "OPC UA Servients"
```

Comment:
Theoretical, this would be possible. However, that should not be the primary goal for now.